client/src/main/scala/HMDAGeoClientService.scala

```diff
@@ -0,0 +1,48 @@
+package hmda.geo.client
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.stream.{ ActorMaterializer, StreamTcpException }
+import scala.concurrent.duration._
+import akka.util.Timeout
+import com.typesafe.config.Config
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model.StatusCodes._
+import akka.http.scaladsl.model._
+import akka.stream.scaladsl.{ Sink, Source }
+import scala.concurrent.{ ExecutionContext, Future }
+import hmda.geo.client.model.ResponseError
+import hmda.geo.client.protocol.ClientJsonProtocol
+
+trait HMDAGeoServiceClient extends ClientJsonProtocol {
```

Comment:
The name of this file is `HMDAGeoClientService`, while this trait is named `HMDAGeoServiceClient`.  _Seems_ like the filename should be changed to match this trait name?